### ISSUE TITLE
Unable to edit chat titles (input doesn't hold keyboard focus)

### ISSUE THREAD

**Issue by aaron-em** [open-webui/open-webui#952]
# Bug Report

## Description

**Bug Summary:**
When attempting to edit a chat title in the sidebar, the `input` element cannot be focused, making it impossible to type anything into the field.

A workaround exists, and is described below.

**Steps to Reproduce:**
1. Create a chat.
2. Open the sidebar.
3. Click the pencil icon to attempt to edit the chat title.

**Expected Behavior:**
1. On clicking the pencil icon, keyboard focus is given to the `input` element containing the chat title.
2. When the `input` element is clicked, keyboard focus remains on it, enabling text to be entered.

**Actual Behavior:**
The `input` element gains and immediately loses focus.

## Environment

- **Operating System:** MacOS 13.5
- **Browser (if applicable):** Firefox 115.0.3; Chrome 122.0.6261.94

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I have reviewed the troubleshooting.md document.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
    _No container logs included as they are not applicable to this frontend issue_

## Logs and Screenshots

**Browser Console Logs:**

With `temp0` as a reference to the `input` element, adding listeners for `focus`, `blur`, and `click` events reflects the element gaining and then immediately losing focus:

```
temp0.addEventListener('focus', () => console.log('focused'))
undefined
temp0.addEventListener('blur', () => console.log('blurred'))
undefined
temp0.addEventListener('click', () => console.log('clicked'))
undefined

// on clicking the input element, the following logs are emitted:

focused [debugger eval code:1:47](chrome://devtools/content/webconsole/debugger%20eval%20code)
clicked [debugger eval code:1:47](chrome://devtools/content/webconsole/debugger%20eval%20code)
blurred [debugger eval code:1:46](chrome://devtools/content/webconsole/debugger%20eval%20code)
```

**Docker Container Logs:**
n/a - everything else works fine! 😄 

**Screenshots (if applicable):**
n/a - will provide a screen recording on request, but the repro steps seem fairly clearcut

## Installation Method

Local Docker build from a checkout at the `v0.1.106` tag, without modifications.

## Additional Information

The following workaround enables updating chat titles on MacOS in Firefox (maybe others, haven't tested):

1. In some other editable text field, enter and then copy the title you want the chat to have.
1. Click the pencil icon to edit.
2. Triple-click the `input` to select all text it contains. **Do not release the mouse button after the third click.**
3. While still holding the mouse button, paste, replacing the entire chat title with the text you copied in step 1.
4. Release the mouse button and save the change.
